sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -3,13 +3,20 @@
 ## 1.0.0-beta.6 (Unreleased)
 
 ### Features Added
+* Added `QueryType` parameter to `AISearchIndexResource` to allow different search types performed by an agent.
+* Added sample for Azure AI Search in streaming scenarios.
 
 ### Breaking Changes
 
 ### Bugs Fixed
+* Fixed Azure AI Search in streaming scenarios.
```

Comment:
Was the fix related url citations specifically? Let's add more information for that